File: gestor-pagos-frontend/src/app/modelos/CrearPromocionDTO.ts
export interface CrearPromocionDTO {
  nombre: string;
  descuentoResidencial: number;
  descuentoEmpresarial: number;
  aplicaNuevos: boolean;
  tipoPromocion: string;
  vigenciaDesde: string;
  vigenciaHasta: string;
  activa: boolean | null;
}

File: ApiIntegrador/Dto/CrearPromocionDTO.cs
namespace ApiIntegrador.Dto
{
    public class CrearPromocionDTO
    {
        public string Nombre { get; set; } = string.Empty;
        public decimal DescuentoResidencial { get; set; }
        public decimal DescuentoEmpresarial { get; set; }
        public bool AplicaNuevos { get; set; }
        public string TipoPromocion { get; set; } = string.Empty;
        public DateTime VigenciaDesde { get; set; }
        public DateTime VigenciaHasta { get; set; }
        public bool Activa { get; set; } = true;
        public int? IdCiudad { get; set; } 
    }
}
